windows-support/Makefile

```diff
@@ -1,5 +1,8 @@
 all: dist/innosetup/vagrant-spk-setup.exe
 
+WINE:=wine
```

Comment:
Stylistically it'd be nice to surround the assignment operator with a ` ` like `x := y`. I can add that in a commit on top.